libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj

```diff
@@ -16,7 +16,7 @@
     <Description>Library for connecting bots with Slack.</Description>
     <Summary>This library implements C# classes for the Slack adapter.</Summary>
     <!-- The SlackAPI package isn't signed, so supress the warning. There seems to not be a way to supress this for ONLY SlackAPI. -->
-    <NoWarn>$(NoWarn),CS8002</NoWarn>
+    <NoWarn>$(NoWarn),CS8002,CS1591</NoWarn>
```

Comment:
>CS1591 [](start = 29, length = 6)

Why?

Reply:
I'll check this again. I was getting build errors due to the Slack package not having code docs which I am addressing in a separate pr. Not sure why the suppression appears twice in project file. Will confirm tomorrow.

Reply:
@mrivera-ms I have now updated this to remove the duplicate suppressions and leave just the individual property group that should be there https://github.com/microsoft/botbuilder-dotnet/blob/25d3c7d3aa9a968affd2fda4b747783aad1998d0/libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj#L28